python/glow/wgr/functions.py

```diff
@@ -157,7 +158,8 @@ def pivot_for_gwas(spark: SparkSession, label_df: pd.DataFrame) -> DataFrame:
         transposed_df = label_df.T
         column_names = ['label', 'values']
     elif label_df.index.nlevels == 2:  # Indexed by sample id and contig name
-        transposed_df = label_df.T.stack()
+        sample_ordering = pd.unique(label_df.index.get_level_values(0))
+        transposed_df = label_df.T.stack().reindex(sample_ordering, axis=1)
```

Comment:
What happens if we use `.reindex(columns=labeldf.index)` here?

Reply:
If the original index was a multi-index on sample id and contig name, so unfortunately that does not work. We need to the `unique` because the sample ids appear for each chromosome.